src/main/java/de/retest/recheck/Properties.java

```diff
@@ -2,7 +2,8 @@
 
 public class Properties {
 
-	public static final String FILE_OUTPUT_FORMAT_PROPERTY = "de.retest.output.Format";
+	public static final String STATE_OUTPUT_FORMAT_PROPERTY = "de.retest.output.Format";
+	public static final String REPORT_OUTPUT_FORMAT_PROPERTY = "de.retest.report.output.format";
```

Comment:
I guess we have to keep `de.retest.output.Format` for backwards compatability, but I would rather have:

* `de.retest.recheck.output.stateFormat`
* `de.retest.recheck.output.reportFormat`

So maybe:

```suggestion
	public static final String REPORT_OUTPUT_FORMAT_PROPERTY = "de.retest.recheck.output.reportFormat";
```